lib/third_party/mcu_vendor/espressif/esp-idf/components/esp32/intr_alloc.c

```diff
@@ -809,10 +809,27 @@ esp_err_t IRAM_ATTR esp_intr_enable(intr_handle_t handle)
     return ESP_OK;
 }
```

Comment:
seeking clarification on whether port_enter_critical is needed in enable as well.